modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_mini/transition.js

```diff
@@ -14,22 +14,20 @@
 
 export default function ({namespace}) {
 	Liferay.provide(window, `${namespace}transition`, (event) => {
-		const link = event.currentTarget;
+		const link = event.delegateTarget;
```

Comment:
Not 100% sure what `event` you have access to here, perhaps it's a YUI event, but if it's a [vanilla JS event](https://developer.mozilla.org/en-US/docs/Web/API/Event) then `delegateTarget` doesn't exist on it

Reply:
Hi @andrea-ale-sbarra  and @kresimir-coko , delegateTarget will exist always if our [view.js](modules/apps/commerce/commerce-cart-content-web/src/main/resources/META-INF/resources/js/cart_mini/view.js) is called.


See: https://github.com/liferay-commerce/liferay-portal/blob/ff484b7f5d6f46c85c6e90a2e607da3a26ad8afd/modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/delegate/delegate.es.js#L55

@andrea-ale-sbarra , let me know if everything is good.

Reply:
IDK the intricacies of your infrastructure, but you linked to a `delegate` utility which needs to be imported and used as specified in the API:
```js
/**
 * Listens to the specified event on the given DOM element, but only calls the
 * given callback listener when it's triggered by elements that match the
 * given selector or target element.
 * @param {!Element} element The DOM element the event should be listened on.
 * @param {string} eventName The name of the event to listen to.
 * @param {string} selector Css selector that should match the event for the
 *     listener to be triggered.
 * @param {!function(!Object)} callback Function to be called when the event
 *     is triggered. It will receive the normalized event object.
 * @return {function} Can be used to remove the listener.
 */
function delegate(element, eventName, selector, callback) {
```

It does not magically enable `event.delegateTarget` to be used wherever in the codebase.

This needs to be manually tested by whoever is in charge.

Reply:
@kresimir-coko , if you look at commit: https://github.com/liferay-commerce/liferay-portal/pull/3161/commits/e247c8c0fcd52b343e12b70436995b3f47b25067 and see the full file, you will already see delegate is being imported and used as specified in the API.

Reply:
Also, I have already manually tested this.

Reply:
👍